.gitignore

```diff
@@ -1,3 +1,4 @@
+.vscode/
```

Comment:
We shouldn't add this, because we have meaningful configuration checked into the repo.